pkgs/by-name/cl/clipse/package.nix

```diff
@@ -0,0 +1,28 @@
+{ 
+  lib, 
+  buildGoModule, 
+  fetchFromGitHub 
```

Comment:
The following formatting is quite popular with nixpkgs:
```suggestion
{ lib 
, buildGoModule
, fetchFromGitHub 
```

Reply:
Thank you for your help @afh :pray: 

I've made the suggested changes now - please let me know if I've missed anything :smiley: